src/DemoUI/src/Services/Integration/UrlService.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Packlink\DemoUI\Services\Integration;
+
+/**
+ * Class UrlService
+ * @package Packlink\DemoUI\Repository
+ */
+class UrlService
+{
+    /**
+     * @param $controllerName
+     * @param $action
+     *
+     * @return string
+     */
+    public static function getEndpointUrl($controllerName, $action)
```

Comment:
Codacy found an issue: [getEndpointUrl accesses the super-global variable $_SERVER.](https://app.codacy.com/gh/packlink-dev/ecommerce_module_core/pullRequest?prid=5826400)